libutils/sequence.c

```diff
@@ -449,3 +450,103 @@ int SeqStringLength(Seq *seq)
 
     return total_length;
 }
+
+#define STR(x) #x
+#define STR_EXPAND(x) STR(x)
+#define SEQ_PREFIX_LEN 10
+
+// TODO: These static helper functions could be (re)moved
+static bool HasNulByte(const char *str, size_t n)
```

Comment:
Missing an L?

Reply:
Haha, @vpodzime ! This is exactly what I said to him, but apparently calling it `NUL` byte and `NULL` pointer is popular (in some circles, I guess).

Reply:
Pretty sure it's called NUL-byte because the ASCII-"names" are limited to three characters.

Reply:
The actual name should probably be HasNullByte still :P

Reply:
@karlhto Not necessarily. If enough people use the term Nul-byte (with one l) it makes sense to call it that. I'm not sure though.

Reply:
We can compromise and you can call it HasNullCharacter